integration_tests/src/lib.rs

```diff
@@ -2628,7 +2628,10 @@ mod test {
         }
 
         // Old volume is no longer active
-        assert!(!volume.query_is_active().await?);
+        while volume.query_is_active().await? {
```

Comment:
I hit this panic while testing, and I think it's just been lucky that we have not seen
it happen before.  Maybe things are faster now, where as before we would
always deactivate before the check happened.